packages/eslint-plugin/src/rules/no-unsafe-member-access.ts

```diff
@@ -86,12 +86,14 @@ export default createRule({
             messageId = 'unsafeThisMemberExpression';
           }
         }
+        const isErrorType = tsutils.isIntrinsicErrorType(type);
 
         context.report({
           node: node.property,
           messageId,
           data: {
             property: node.computed ? `[${propertyName}]` : `.${propertyName}`,
+            type: isErrorType ? '`error`' : '`any`',
```

Comment:
#8304 went with:

```suggestion
            type: isErrorType ? '`error` typed' : '`any`',
```

Any reason not to do that here?

Reply:
It was omitted🙇‍♂️ Fixed in 866d68d.